engine/classes/ElggInstaller.php

```diff
@@ -356,6 +356,12 @@ protected function database($submissionVars) {
 				'value' => 'elgg_',
 				'required' => TRUE,
 				),
+			'timezone' => array(
+				'type' => 'dropdown',
+				'value' => 'UTC',
+				'options' => \DateTimeZone::listIdentifiers(),
+				'required' => TRUE
+			)
```

Comment:
This doesn't prevent such user specific timezone support, this just sets the initial server default timezone that the site is operating in as required for php 5.5+.  I agree that most times people would want it standardized at UTC, which is why I made it the default, but left it configurable just in case (and even if they change it at installation time, as long as it stays consistent there should be no issues).  Take a look at the arck events plugin to see how we handle user defined timezones.  That is currently working even while having the default timezone set in the settings.php